main.go

```diff
@@ -90,14 +91,14 @@ func (c *collector) Collect(ch chan<- prometheus.Metric) {
 	metrics, err := c.queryOpenTSDB()
 	if err != nil {
 		log.Errorf("Error scraping target %s: %s", c.target, err)
-		ch <- prometheus.NewInvalidMetric(prometheus.NewDesc("api_error", "Error scraping target", nil, nil), err)
+		ch <- prometheus.NewInvalidMetric(prometheus.NewDesc("api_error", "", nil, nil), errors.New("Error scraping target, check exporter logs"))
```

Comment:
@secumod: The intent of this PR is prevent sensitive information from being exposed in the error message that is returned in the HTTP response. We intentionally want it to be different to what's logged to stdout.